database/enums/enum_assessment_grouping.pg

```diff
@@ -0,0 +1 @@
+set, unit
```

Comment:
Not saying it's incorrect, just raising the issue: most non-techinical enums use CamelCase for items, would it be more consistent to have: 
```suggestion
Set, Unit
```

Reply:
You're right, added.